hashring_test.go

```diff
@@ -140,7 +135,7 @@ func (s *RingTestSuite) TestLookup() {
 
 	s.ring.RemoveServer("server1")
 	s.ring.RemoveServer("server2")
-	s.ring.RemoveServer(s.ring.ringpop.node.Address())
+	s.ring.RemoveServer(s.ringpop.WhoAmI())
```

Comment:
the signature of `WhoAmI` has changed to `func () (string, error)` so this needs to be updated.